.github/workflows/scheduled-event-processor.yml

```diff
@@ -20,6 +20,8 @@ permissions: {}
 
 jobs:
   event-handler:
+    # prevents the job from being executed on forks
+    if: github.repository == 'azure/azure-sdk-for-net'
```

Comment:
Unfortunately, this won't work as-is.   The file is pushed out from the tools repository into each of the other SDK repositories and the content can't change.

I wonder, though if we could do some kind of `startswith` for `azure/`?   @JimSuplizio, thoughts?